README

```diff
@@ -0,0 +1,41 @@
+# Ngrok Agent Docker Image
+
+## Usage
+
+Run an ngrok agent pointed at localhost:80
+
+```bash
+docker run ngrok/ngrok http 80
```

Comment:
We might want to file an issue for making it so you can do something like `--net=host -e "INTROSPECT_ADDR=localhost" or such, since otherwise there's a security concern of the web addr being `0.0.0.0` in `net=host`

Reply:
fixed, issue here: #2